bin/smoke

```diff
@@ -216,8 +224,7 @@ end
 
 begin
   @options, @args = parse_options
-  application = @args[0]
-  validate_application(application)
+  application = @args[0].split(/ /) # quick hack, does not consider blanks in file names
```

Comment:
There must be a better way of doing this. I expect there's a way to parse strings with spaces and quotes in the same fashion as the OS.

For example:

    echo 'foo bar baz'

should split to:

    ['echo', 'foo bar baz']